client/state/data-layer/wpcom/sites/comments/likes/mine/delete/test/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_UNLIKE,
+	COMMENTS_LIKE,
+	NOTICE_CREATE,
+} from 'state/action-types';
+import {
+	unlikeComment,
+	updateCommentLikes,
+	handleUnlikeFailure,
+} from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const SITE_ID = 91750058;
+const POST_ID = 287;
+
+describe( '#unlikeComment()', () => {
+	const action = {
+		type: COMMENTS_UNLIKE,
+		siteId: SITE_ID,
+		postId: POST_ID,
+		commentId: 1
+	};
+
+	it( 'should dispatch a http action to remove a comment like', () => {
+		const dispatch = spy();
+		unlikeComment( { dispatch }, action );
+
+		expect( dispatch ).to.have.been.calledOnce;
+		expect( dispatch ).to.have.been.calledWith( http( {
+			apiVersion: '1.1',
+			method: 'POST',
+			path: `/sites/${ SITE_ID }/comments/1/likes/mine/delete`,
+			onSuccess: action,
+			onFailure: action,
+			onProgress: action,
```

Comment:
I'm guessing there's a reason to repeat `action` here instead of passing it as the second param to `http()`? or was it just a slip?